examples/with-svelte/README.md

```diff
@@ -0,0 +1,24 @@
+# Turborepo svelte starter
+
+This is an official starter turborepo.
```

Comment:
```suggestion
This is an official starter Turborepo.
```

Reply:
These are like this in the rest of the example README's so I'll update them everywhere.